nilearn/decoding/space_net.py

```diff
@@ -34,7 +34,7 @@
 from sklearn.linear_model.base import _preprocess_data as center_data
 from .._utils.compat import _basestring
 from .._utils.cache_mixin import CacheMixin
-from .objective_functions import _unmask
+from nilearn.masking import _unmask_from_to_3d_array
```

Comment:
Could you use relative imports please ? Like the ones above and below.

Reply:
@GaelVaroquaux  and I talked about this in #1919 and  while he doesn't want existing imports to be changes, for new imports absolute imports are okay.
PEP8 standard is to use Absolute imports. The only time relative imports are considered okay if  a specific absolute import is too verbose.

Reply:
+1 for relative imports again.

Either we do this or change everything to absolute imports.

Reply:
Given the recommendations of PEP8, we should gradually evolve our code to absolute imports.

Reply:
Are the recommendations changed since last year ? This is what I heard when Loic was there.